f2/src/WhatHappenedPage.js

```diff
@@ -10,58 +10,61 @@ import { Steps } from './components/stepper'
 import { TrackPageViews } from './TrackPageViews'
 import { WhatHappenedForm } from './forms/WhatHappenedForm'
 import { Layout } from './components/layout'
-import { getDoneForms } from './utils/queriesAndMutations'
 import { BackButton } from './components/backbutton'
 import { Stack } from '@chakra-ui/core'
+import { useStateValue } from './utils/state'
 
-export const WhatHappenedPage = () => (
-  <Route
-    render={({ history }) => (
-      <Layout>
-        <TrackPageViews />
-        <Stack spacing={10} shouldWrapChildren>
-          <BackButton route="/timeframe">
-            <Trans id="whatHappendPage.backButton" />
-          </BackButton>
+export const WhatHappenedPage = () => {
+  const [data, dispatch] = useStateValue()
+  const { doneForms } = data
 
-          <Stack spacing={4} role="heading" aria-level="1">
-            <Steps activeStep={2} totalSteps={6} />
-            <H1 as="span">
-              <Trans id="whatHappendPage.title" />
-            </H1>
-          </Stack>
-          <Stack spacing={4}>
-            <P>
-              <Trans id="whatHappendPage.intro1" />
-            </P>
+  return (
+    <Route
+      render={({ history }) => (
+        <Layout>
+          <TrackPageViews />
+          <Stack spacing={10} shouldWrapChildren>
+            <BackButton route="/timeframe">
+              <Trans id="whatHappendPage.backButton" />
+            </BackButton>
 
-            <P>
-              <Trans id="whatHappendPage.details1" />
-            </P>
+            <Stack spacing={4} role="heading" aria-level="1">
+              <Steps activeStep={2} totalSteps={6} />
+              <H1 as="span">
+                <Trans id="whatHappendPage.title" />
```

Comment:
Typo in 'whatHappendPage'  (should be whatHappenedPage), same typo reflected in en.json.

Reply:
good catch! let's add an issue to correct this spelling - it's all over the place (cut and paste FTL)